Description: Remove obsolete non-standard resetterm calls.
 Source commits: 26722b82bb7cbc1141a0621ba1d870e8f0c26e27
Origin: upstream
Forwarded: yes
Last-Update: 2026-07-13

diff --git a/Wnn/conv/cvt_read.c b/Wnn/conv/cvt_read.c
index 6ad3255..f65e38b 100644
--- a/Wnn/conv/cvt_read.c
+++ b/Wnn/conv/cvt_read.c
@@ -644,11 +644,9 @@ convert_getterm (termname, flg)
   if (rsl != 1)
     {
       errdsp2 ("Can't get terminfo entry", termname);
-      /* このときresetterm()は必要か？ */
       return (1);
     }
   convert_getstrs (flg);
-  resetterm ();
   return (0);
 }
 #endif
diff --git a/Wnn/uum/termio.c b/Wnn/uum/termio.c
index cf23998..f85133f 100644
--- a/Wnn/uum/termio.c
+++ b/Wnn/uum/termio.c
@@ -183,7 +183,6 @@ openTermData ()
 void
 closeTermData ()
 {
-  resetterm ();
   reset_shell_mode ();
 }
 
